components/ChooseSong.js

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react'
+import { Container, Row, Col, Table, Form } from 'react-bootstrap/'
+import PropTypes from 'prop-types'
+import { searchSpotifyTracks } from '../utils/fetcher'
+import useAuth from '../utils/useAuth'
+const AddMemory = () => {
```

Comment:
Would chuck in a space between the imports and everything else. Plus I'd change AddMemory to ChooseSong

Reply:
Done!